### DISCUSSION TITLE
chezmoi try to decrypt files although they in `.chezmoiignore`

### DISCUSSION THREAD

**Discussion by unavaliabl3** [twpayne/chezmoi#2091] [Q&A] [answered]
I followed this [comment](https://github.com/twpayne/chezmoi/issues/927#issuecomment-722029861) in #927 but chezmoi still try to decrypt ignored files and show this error:

```
age: error: no identity matched any of the recipients
age: report unexpected or unhelpful errors at https://filippo.io/age/report
chezmoi: exit status 1
```

`chezmoi.toml`:
```toml
encryption = "age"
[data]
trusted = false
```

**Comment by twpayne** [comment, marked as answer]
Please show how to reproduce the problem.

**Reply by unavaliabl3** [reply]
Sorry for wasting your time. I made a typo in `.chezmoiignore`. Should have checked the output of `chezmoi ignored` before posting.
